AppAuth-JS/src/flags.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017 Google Inc.
```

Comment:
Licence

Reply:
We'll keep the Google's license, since I did not change anything in here.

Reply:
@kasunbg can you please explain the practice in this case?

Reply:
@thanujalk If no lines were changed, then it is ok to keep the copyright as it is. We followed that practice for wso2-axis2 repo for example.

https://github.com/wso2/wso2-axis2/blob/master/modules/transport/http/src/org/apache/axis2/transport/http/AbstractAgent.java#L2